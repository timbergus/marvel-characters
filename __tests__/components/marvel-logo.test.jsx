import React from 'react';
import { shallow } from 'enzyme';

import MarvelLogo from '../../src/components/marvel-logo';

describe('Loader', () => {
  test('should render the marvel logo.', () => {
    const component = shallow(<MarvelLogo />);
    expect(component).toMatchSnapshot();
  });
});
