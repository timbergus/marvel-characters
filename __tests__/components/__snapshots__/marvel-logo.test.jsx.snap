// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render the marvel logo. 1`] = `ShallowWrapper {}`;
